cases/forms/reporting.py

```diff
@@ -153,10 +153,17 @@ class WhereForm(StepForm):
         label="Is the residence a Hackney Estates property?",
         widget=forms.RadioSelect,
         required=False,
-        help_text="Optional",
         choices=Case.ESTATE_CHOICES,
     )
 
+    def clean(self):
+        where = self.cleaned_data.get("where")
+        estate = self.cleaned_data.get("estate")
+        if where == "residence" and not estate:
+            self.add_error(
+                "estate", forms.ValidationError("Please answer the question")
```

Comment:
I have a feeling this error message should give a hint as to what "the question" is, because the message will be displayed in a red box at the top of the page – not right next to the form input.

(See other examples in `ValidationError` examples in this file.)

Reply:
Yeah, sorry, was lazy